### PR TITLE
Substantial revisions to adapt to Wikitext Extension for VSCode

### PR DESCRIPTION
**Pull request by Frederisk**
- Ignore more source code when packaging.
- Remove client.
- Rename the package.
- Adjusted the description of some settings.

### CODE REVIEW

File: LICENSE.txt
Comment (bhsd-harry) [inline LICENSE.txt:1]: Is this necessary?

File: .gitignore
Comment (bhsd-harry) [inline .gitignore:1]: Please ignore `.vscode` and all yarn-related files.

**Review by bhsd-harry — CHANGES_REQUESTED** [review]

File: .gitignore
Comment (Frederisk) [reply .gitignore:1]: `.vscode` should not be ignored, this is not a temporary folder, it is used to store valuable information such as debugging configurations. Maybe you can look at other repositories and they do not ignore this file. Such as [CMake](https://github.com/microsoft/vscode-cmake-tools/tree/main/.vscode).

`.yarnrc` is also not a temporary file, this file stores flags used to suppress a check on it against the engine. This is [a legacy issue](https://github.com/microsoft/vscode/issues/91009#issuecomment-588908757) from the classic version.

File: LICENSE.txt
Comment (Frederisk) [reply LICENSE.txt:1]: Not required, but would be nice. VSCode treats it as MarkDown by default, which is a bit annoying. In addition, `vsce` will always add the extension name of `.txt` to this file after packaging.

Comment (bhsd-harry) [reply LICENSE.txt:1]: In this case, please rename it back.

File: .gitignore
Comment (bhsd-harry) [reply .gitignore:1]: Then please at least ignore `.vscode/settings.json`.

**Review by bhsd-harry — CHANGES_REQUESTED** [review]

**Pull request merged**